protected/tests/unit/models/AUSPatientTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: root
+ * Date: 22/09/17
+ * Time: 2:11 PM
+ */
+
+class AUS_PatientTest extends CDbTestCase
+{
+    protected $fixtures = array('patient' => 'Patient');
+    public function setUp()
+    {
+        parent::setUp();
```

Comment:
You might not need these if they are just calling the parents, although they might be useful later...

Reply:
I'd say leave them in for the time being. There is every chance we'll need to add more to these functions as more test cases are added.